.github/workflows/presubmit.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
   build-windows:
     name: build-windows
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
Strictly speaking no, that piece of code is for their CI only, doesn't affect our local builds. It means that the Github Actions workflow runs on an agent with Windows Server 2019 installed.

Reply:
The compiler error you found was because `#inlcude <type_traits>` was missing from Handle.h, thus `std::enable_if_t` was undefined. Luckily they fix it right in the next version: v1.22.2. So I'd not bother fixing it as part of this PR, but let the next one fix it.